src/ui/components/VPNToggle.qml

```diff
@@ -12,6 +12,12 @@ VPNButtonBase {
     id: toggleButton
 
     property var toggleColor: Theme.vpnToggleDisconnected
+    property var describedBy: logoTitle.text
+    property var labelledBy: logoSubtitle.text
+    property var toolTipTitle: ""
+
+
+    Accessible.name: VPNController.state === VPNController.StateOn ? describedBy + ". " + toolTipTitle : describedBy + ". " + labelledBy + "."
```

Comment:
@yzen  After trying this with the default mac screen reader, I left `labelledBy` out of the `Accessible.name` when the VPN is connected because it resets itself every second and gets very noisy and difficult to understand.

Reply:
Thanks for testing it with VO :). I tried as well and it still feels a little verbose. I think I'm gonna take back my comment about needing both label and description here as the description does not really add much value.

How about we just keep:
```
property var toolTipTitle: ""
Accessible.name: toolTipTitle
```

and avoid setting labelledBy and describedBy properties.